### PR TITLE
Added tableschema-sql, pybloom_live

### PR DESCRIPTION
**Pull request by roll**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

---

- fixes https://github.com/frictionlessdata/tableschema-sql-py/issues/79

### CODE REVIEW

**Comment by roll** [comment]
Hi @conda-forge/help-python, please take a look

File: recipes/pybloom_live/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 837550da7c82adb14b901c56c3e131ad269250cd19850a53671e878bd0626c68
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (marcelotrevisani) [inline recipes/pybloom_live/meta.yaml]: Could it be ``noarch`` ?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```

File: recipes/pybloom_live/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    - bitarray >=0.3.4
+
+test:
+  imports:
+    - pybloom_live
+
+about:
+  home: https://github.com/joseph-fox/python-bloomfilter
+  license: MIT
+  license_family: MIT
```
Comment (marcelotrevisani) [inline recipes/pybloom_live/meta.yaml]: Please add the ``license_file``
You will have to package this license manually. 

Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html#packaging-the-license-manually

File: recipes/tableschema-sql/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 524d5aa4da2fad6a1fd6a455a2c4727f794dcd25b69e765731c2e106d47a664d
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```
Comment (marcelotrevisani) [inline recipes/tableschema-sql/meta.yaml]: Could it be ``noarch`` as well?
```suggestion
  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
  noarch: python
```

**Comment by roll** [comment]
Thanks @marcelotrevisani 

Please review, I've updated

**Comment by roll** [comment]
Thanks!

**Pull request merged**